### PR TITLE
Improvements and fixes before initial release

### PR DESCRIPTION
**Pull request by mecha**
:warning: Requires the corresponding [interface package][1] to be released first.

[1]: https://github.com/RebelCode/wp-nonce-interface

### CODE REVIEW

File: src/Factory/NonceFactory.php [hunk 1]
```diff
@@ -39,4 +39,14 @@ protected function _createNonceInstance($id, $code)
     {
         return new Nonce($id, $code);
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    protected function _generateNonceCode($id)
```
Comment (XedinUnknown) [inline src/Factory/NonceFactory.php:48]: `AbstractNonceFactory`, the ancestor of this class, already [implements](https://github.com/RebelCode/wp-nonce/blob/develop/src/Factory/AbstractNonceFactory.php#L50) this method.

File: src/Nonce.php [hunk 2]
```diff
@@ -42,4 +44,14 @@ public function getCode()
     {
         return $this->_getCode();
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    public function __toString()
```
Comment (XedinUnknown) [inline src/Nonce.php:53]: Very nice!

File: src/Factory/NonceFactory.php (hunk 1 above)
Comment (mecha) [reply src/Factory/NonceFactory.php:48]: That's on `develop`. See [`AbstractNonceFactory`][1] on this PR's branch.

[1]: https://github.com/RebelCode/wp-nonce/blob/task/improvements/src/Factory/AbstractNonceFactory.php

File: src/Nonce.php (hunk 2 above)
Comment (mecha) [reply src/Nonce.php:53]: :)

File: src/Factory/NonceFactory.php (hunk 1 above)
Comment (XedinUnknown) [reply src/Factory/NonceFactory.php:48]: Yep, you're right. And this is correct: WP specifics should be in concrete. <3

**Pull request merged**